src/public/dashboard/dashboard.php

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+ 
+	<title></title>
+	
+	<style>
+ .main-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 500px;
+  width: 900px;
+  margin: 0 auto;
+  border-radius: 25px;
+  background-color: #f0f0f0;
+}
+.main-div h1{
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.row {
+  display: flex;
+   margin-bottom: 40px;
+}
+
+.row1 .rectangle-div{
+  width: 250px;
+  height: 150px;
+  margin-top: 40px;
+  margin-right: 10px;
+  margin: 20px;
+  border-radius: 25px;
+  background-color: black;
+ 
+  transition: transform 0.2s, box-shadow 0.2s;
+}
+
+.rectangle-div:hover{
+  box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
+
+}
+
+.hover-text{
+   position: absolute;
+  top: 0;
+  left: 0;
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 0.2s, visibility 0.2s;
+}
+.rectangle-div:hover .hover-text{
+  opacity: 1;
+  visibility: visible;
+   position: absolute;
+  top: 50%; /* Center text vertically */
+  left: 50%; /* Center text horizontally */
+  transform: translate(-50%, -50%);
+}
+
+.row2 .square-div {
+  width: 100px;
+  height: 100px;
+  margin-right: 10px;
+  border-radius: 25px;
+  margin: 25px;
+  background-color: black;
+justify-content: center;
+align-content: center;
+display: flex;
+
+}
+svg {
+  display: block;
+  margin: 0 auto;
+  margin-top: 10px;
+}
+.square-div:hover{
+  box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
+
+}
+
+
+
+    </style>
+
+</head>
+<body>
+	
+	  
+</head>
+<body>
+   <div class="main-div">
+    <h1>Dashboard</h1>
+    <div class="row row1">
+      <a href="/catalog/products">
+      <div class="rectangle-div">
+        <?php 
+         require_once DATABASE . '/connect.php';
+
+        $sql = "SELECT COUNT(name) as aantal FROM products";
+        $aantalP = fetch($sql);
+      
+        ?>
+        <p class="font-bold mt-5 ml-3 justify-center flex">Jouw producten: </p>
+        <p class="font-bold mt-0 ml-3 justify-center flex text-2xl mt-6"><?php echo $aantalP['aantal']; ?></p>
+        <a class="hover-text"></a>
+      </div>
+      </a>
+      <div class="rectangle-div">
+      <p class="font-bold mt-5 ml-3 justify-center flex">Jouw verkochte producten: </p>
+        <p class="font-bold mt-0 ml-3 justify-center flex text-2xl mt-6">0</p>
+      </div>
+      <div class="rectangle-div">
+      <p class="font-bold mt-5 ml-3 justify-center flex">Jouw winst: </p>
+        <p class="font-bold mt-0 ml-3 justify-center flex text-2xl mt-6">0</p>
+        <a class="hover-text"></a>
```

Comment:
All this doesn't make sense for the point of view of a buyer. A buyer can't have products, sell products, have profit etc...